addons/website_mail_channel/views/snippets/s_channel.xml

```diff
@@ -13,8 +13,8 @@
         </div>
         <p class="js_mg_details d-none">
             <span class="js_mg_email d-none"><a href="#" class="js_mg_email"><i class="fa fa-envelope-o"/> send mail</a> - </span>
-            <a href="#" class="js_mg_link"><i class="fa fa-file-o"/> archives</a> -
-            <a role="button" href="#" class="js_unfollow_btn"><i class="fa fa-times"/> unsubscribe</a>
+            <a href="#" class="btn btn-primary js_mg_link"><i class="fa fa-file-o"/> archives</a> -
+            <a role="button" href="#" class="btn btn-primary js_unfollow_btn"><i class="fa fa-times"/> unsubscribe</a>
```

Comment:
Strange I have the feeling I already reviewed this PR with the same comments but I must be dreaming. Anyways: what does that have to do with the current PR? :thinking:

Reply:
@qsm-odoo, actually while subscribing to the channel it shows only `<a>` without style which looks weird hence I have applied `btn btn-primary`.

**Before**
![unsub_n_archive](https://github.com/odoo/odoo/assets/84001602/2f518643-67ce-4b41-9fe8-a72f4a2cc2f0)

**After**
![btn_unsub_archive](https://github.com/odoo/odoo/assets/84001602/75d32978-b704-49df-8578-0fe72912ba50)

Also, changes have been applied so that `d-none` gets added at destroy of this public widget.

Reply:
Well, that's the design of the snippet, that's clearly not something we change in a stable version, and if we want that in master, it should be in its own commit and validated with the PO in the related task :)

Reply:
Okay @qsm-odoo, gotcha.
No need to do that in master as we have changed the design to:

![master_unsub](https://github.com/odoo/odoo/assets/84001602/dff40dbc-8467-4871-bbbb-b28823f83cca)

Reply:
Hello @qsm-odoo, changes have been applied.